### PR TITLE
Lesson 02

### PR DESCRIPTION
**Pull request by georgeme**
Index page and CSS (including variables) for Lesson 02 Assignment

### CODE REVIEW

File: lesson02/assignment/css/_components.content.scss [hunk 1]
```diff
@@ -2,16 +2,158 @@ html {
...
-*:before {
-    box-sizing: inherit;
+// Creating the flex container as a row, using reusable variables in case we were to add to this single employee card later
+.container {
+    // padding: 2rem;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+    width: 60%;
```
Comment (brycebenson) [inline lesson02/assignment/css/_components.content.scss:12]: I would give a component with a known dimension (here we know the size of the card's width) an explicit maximum width. So you could just swap that to `width: 100%;` and `max-width: XXpx;` so it still breaks down as expected but doesn't grow too large.-

**Review by brycebenson — COMMENTED** [review]
Great work on this.

File: lesson02/assignment/css/_components.content.scss (hunk 1 above)
Comment (georgeme) [reply lesson02/assignment/css/_components.content.scss:12]: I thought we weren't supposed to use pixels but should use rems and % where possible for responsive? I used 16 px to calculate rems for this. I'm happy to change it to pixels but wondered about previous teachings.

Comment (georgeme) [reply lesson02/assignment/css/_components.content.scss:12]: Actually, I think I get it. The width 100% takes care of the fact that we've specified a max-width.

I think.

Comment (brycebenson) [reply lesson02/assignment/css/_components.content.scss:12]: You want percentages for proportions, but for something like a container or component's max-width it's normal to use pixels.

**Pull request merged**